_posts/2015-07-13-Advances-in-Modal-Logic-2016.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Advances in Modal Logic 2016"
+shorttitle: AIML'16 
+author: "Valentin Montmirail"
```

Comment:
I'm not sure here. 

The documentation about how to contribute says "your name", 
but it doesn't seems logical that my name appears on a news who have no link with me.

If this `author` should not be me, please replace with [Roman Kontchakov](mailto:roman@dcs.bbk.ac.uk)